README.md

```diff
@@ -1,25 +1,53 @@
-# Fullstack Boilerplate
+# Fullstack Boilerplate Sanic/React
 
-Use it for setting up a take-home interview assignment or a portfolio project.
+A backbone for your coding challenge.
```

Comment:
Sounds very poetic 😄 
Just curious, did we use it from another boilerplate template?

Reply:
ye, borrowed it from another challenge